Tasks/PublishCodeCoverageResults/publishcodecoverageresults.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
+
+import tl = require('vsts-task-lib/task');
+
+var codeCoverageTool = tl.getInput('codeCoverageTool', true);
+var summaryFileLocation = tl.getInput('summaryFileLocation', true);
+var reportDirectory = tl.getInput('reportDirectory');
+var additionalCodeCoverageFiles = tl.getInput('additionalCodeCoverageFiles');
+
+tl.debug('codeCoverageTool: ' + codeCoverageTool);
+tl.debug('summaryFileLocation: ' + summaryFileLocation);
+tl.debug('reportDirectory: ' + reportDirectory);
+tl.debug('additionalCodeCoverageFiles: ' + additionalCodeCoverageFiles);
+
+if(additionalCodeCoverageFiles && (additionalCodeCoverageFiles.indexOf('*') >= 0 || additionalCodeCoverageFiles.indexOf('?') >= 0)) {
```

Comment:
? and * mark are the regex characters. If they are present in the additionalCoverageFiles we try to fliter them. If not we directly add it to array assuming it is a file name without any regex chars